src/Compilers/CSharp/Test/Syntax/Parsing/StatementAttributeParsingTests.cs

```diff
@@ -5983,20 +5983,20 @@ void Goo(int a, int b)
                             N(SyntaxKind.OpenBraceToken);
                             N(SyntaxKind.ExpressionStatement);
                             {
-                                N(SyntaxKind.AttributeList);
```

Comment:
this is another expected change.  we now see this as an invalid range operation where the LHS is a collection literal vs an attribute.